app/src/interfaces/input-rich-text-md/input-rich-text-md.vue

```diff
@@ -104,26 +131,34 @@
 				</template>
 			</v-menu>
 
-			<v-button v-tooltip="t('wysiwyg_options.image')" small icon @click="imageDialogOpen = true">
-				<v-icon name="insert_photo" />
-			</v-button>
-
 			<v-button
-				v-for="custom in customSyntax"
-				:key="custom.name"
-				v-tooltip="custom.name"
+				v-if="view[0] !== 'preview' && toolbar.indexOf('image') !== -1"
+				v-tooltip="t('wysiwyg_options.image')"
 				small
 				icon
-				@click="edit('custom', custom)"
+				@click="imageDialogOpen = true"
 			>
-				<v-icon :name="custom.icon" />
+				<v-icon name="insert_photo" />
 			</v-button>
 
+			<span v-if="view[0] !== 'preview'">
+				<v-button
+					v-for="custom in customSyntax"
+					:key="custom.name"
+					v-tooltip="custom.name"
+					small
+					icon
+					@click="edit('custom', custom)"
+				>
+					<v-icon :name="custom.icon" />
+				</v-button>
+			</span>
+
 			<div class="spacer"></div>
 
 			<v-item-group v-model="view" class="view" mandatory rounded>
-				<v-button x-small value="editor">Editor</v-button>
-				<v-button x-small value="preview">Preview</v-button>
+				<v-button x-small value="editor" :class="[{ active: view[0] !== 'preview' }]">Edit</v-button>
```

Comment:
Since we're here, l think we can add translation strings to "Edit" and "Preview" as well, just for completeness sake in terms of our ongoing effort of whole app translations.

Reply:
Thanks for your feedback!
I've made a commit that should resolve all these requests.